e2e/tests/core/03-connection/connection_test.go

```diff
@@ -30,19 +29,11 @@ type ConnectionTestSuite struct {
 
 // QueryMaxExpectedTimePerBlockParam queries the on-chain max expected time per block param for 03-connection
 func (s *ConnectionTestSuite) QueryMaxExpectedTimePerBlockParam(ctx context.Context, chain ibc.Chain) uint64 {
-	queryClient := s.GetChainGRCPClients(chain).ParamsQueryClient
-	res, err := queryClient.Params(ctx, &paramsproposaltypes.QueryParamsRequest{
-		Subspace: ibcexported.ModuleName,
-		Key:      string(connectiontypes.KeyMaxExpectedTimePerBlock),
-	})
-	s.Require().NoError(err)
-
-	// removing additional strings that are used for amino
-	delay := strings.ReplaceAll(res.Param.Value, "\"", "")
-	time, err := strconv.ParseUint(delay, 10, 64)
+	queryClient := s.GetChainGRCPClients(chain).ConnectionQueryClient
+	res, err := queryClient.ConnectionParams(ctx, &connectiontypes.QueryConnectionParamsRequest{})
```

Comment:
Don't we need to support querying the legacy x/params module for older versions? i.e. use a feature releases matrix to toggle between querying using the `ConnectionQueryClient` and the `ParamsQueryClient`

Reply:
ah yea, I think you're right

Reply:
I think it would suffice to go with the same feature release for this (`IsSelfParams`). ConnectionParams was added with https://github.com/cosmos/ibc-go/pull/2745 and I don't see much benefit in adding an additional feature release var for it?